runtime/src/topo/hwloc/topo-hwloc.c

```diff
@@ -533,7 +534,6 @@ void chpl_topo_interleaveMemLocality(void* p, size_t size) {
   obj = hwloc_get_root_obj(topology);
   set = hwloc_bitmap_dup(obj->cpuset);
 
-  flags = HWLOC_MEMBIND_MIGRATE | HWLOC_MEMBIND_STRICT;
   CHK_ERR_ERRNO(hwloc_set_area_membind(topology, p, size, set, HWLOC_MEMBIND_INTERLEAVE, flags) == 0);
```

Comment:
Surely we want to pass `0` for the flags arg here, rather than an indeterminate value.  😁

Reply:
Updated